.storybook/pages/AlongDemo/AlongDemo.tsx

```diff
@@ -28,53 +28,63 @@ import MicrosoftLogo from '../../static/microsoft-logo.svg';
 import Sprout from '../../static/sprout.svg';
 
 const GlobalFooter = ({ className }: { className?: string }) => (
-  <div className={clsx(styles['wireframe-demo__footer'], className)}>
-    <Link>
-      Privacy Policy{' '}
-      <Icon
-        name="open-in-new"
-        purpose="informative"
-        size="0.8em"
-        title="opens in a new tab"
-      />
-    </Link>
-    <Link>Cookie Settings</Link>
-    <Link>
-      User Agreement{' '}
-      <Icon
-        name="open-in-new"
-        purpose="informative"
-        size="0.8em"
-        title="opens in a new tab"
-      />
-    </Link>
-    <Link>
-      Code of Conduct{' '}
-      <Icon
-        name="open-in-new"
-        purpose="informative"
-        size="0.8em"
-        title="opens in a new tab"
-      />
-    </Link>
-    <Link>
-      Help Center{' '}
-      <Icon
-        name="open-in-new"
-        purpose="informative"
-        size="0.8em"
-        title="opens in a new tab"
-      />
-    </Link>
-  </div>
+  <ul className={clsx(styles['wireframe-demo__footer'], className)}>
+    <li className={styles['wireframe-demo__footer-link']}>
+      <Link>
+        Privacy Policy{' '}
+        <Icon
+          name="open-in-new"
+          purpose="informative"
+          size="0.8em"
+          title="opens in a new tab"
+        />
+      </Link>
+    </li>
+    <li className={styles['wireframe-demo__footer-link']}>
+      <Link>Cookie Settings</Link>
+    </li>
+    <li className={styles['wireframe-demo__footer-link']}>
+      <Link>
+        User Agreement{' '}
+        <Icon
+          name="open-in-new"
+          purpose="informative"
+          size="0.8em"
+          title="opens in a new tab"
+        />
+      </Link>
+    </li>
+    <li className={styles['wireframe-demo__footer-link']}>
+      <Link>
+        Code of Conduct{' '}
+        <Icon
+          name="open-in-new"
+          purpose="informative"
+          size="0.8em"
+          title="opens in a new tab"
+        />
+      </Link>
+    </li>
+    <li className={styles['wireframe-demo__footer-link']}>
+      <Link>
+        Help Center{' '}
+        <Icon
+          name="open-in-new"
+          purpose="informative"
+          size="0.8em"
+          title="opens in a new tab"
+        />
+      </Link>
+    </li>
+  </ul>
 );
 
 const LoggedOutPage = ({ onLogin }: { onLogin: () => void }) => (
   <div className={styles['logged-out-page']}>
     <div className="flex flex-col items-center mb-16 p-8 grow">
       <header className="flex gap-4">
         <img
-          alt="placeholder for decorative illustration"
+          alt="A studious youth with specs smiling and looking at his laptop"
```

Comment:
These alt values could also be empty because they're decorative. I figure for the purpose of the demos, it makes sense to indicate that there are now actual branded images whereas in the wireframe demo they're placeholders. I copied the alt text from the along page.